react/components/EXPERIMENTAL_Table/context/body.tsx

```diff
@@ -19,7 +19,20 @@ export function useBodyContext() {
 
 export function BodyProvider({
   children,
-  ...value
```

Comment:
This was re-creating the value of the context everytime something changed inside the `Table`'s `children`